tabpy/tabpy_server/app/app.py

```diff
@@ -160,7 +160,7 @@ def _parse_config(self, config_file):
         self.python_service = None
         self.credentials = {}
 
-        parser = configparser.ConfigParser()
+        parser = configparser.ConfigParser(os.environ)
```

Comment:
In `set_parameter` we specifically check for environment variables with the same names and config file parameters. Do we remove that check now?

Reply:
It looks to me like we only look for environment variables if the parameter is not set in the config.

https://github.com/tableau/TabPy/blob/dev/tabpy/tabpy_server/app/app.py#L179

Reply:
Misunderstood your comment - as a side effect of my change we no longer need to check for environment variables.

Removed the additional check.